pkg/reconciler/ingress/controller.go

```diff
@@ -49,22 +54,30 @@ const (
 	gatewayLabelKey   = "app"
 	gatewayLabelValue = "3scale-kourier-gateway"
 
-	nodeID         = "3scale-kourier-gateway"
-	gatewayPort    = 19001
-	managementPort = 18000
+	nodeID             = "3scale-kourier-gateway"
+	gatewayPort        = 19001
+	managementPort     = 18000
+	cacheWarmUPTimeout = 720 * time.Second
```

Comment:
This seems to be a long timeout. How many ingresses should kourier need to reconcile to take more than this? :thinking:

Reply:
The tests show that resyncing  ~1000 ksvcs could take a long time.